avocado-setup.py

```diff
@@ -609,6 +609,9 @@ def parse_test_config(test_config_file, avocado_bin, enable_kvm):
     parser.add_argument('--enable-kvm', dest="enable_kvm", action='store_true',
                         default=False, help='enable bootstrap kvm tests')
 
+    parser.add_argument('--enable-prepost', dest="enable_prepost",
```

Comment:
@PraveenPenguin I am of a slightly different opinion here.
My thought is to have same wrapper command line for all runs, rather differentiate the need for pre/post scripts by having the folder populated with scripts. This method has better controls according to me.

Suggestions welcome.

Reply:
I would even have this controlled via test cfg files, if possible

Reply:
i think this way as we are using a local copy of wrapper and it used for distro as well IO so we can have script placed in a predefined folder but in one case we don't want to run those script and another case we want to if a directory is there also we can have control and in same copy how we handle dynamic way i am not clear so it seems me the easy way as accepted as well so thought if you think other wise let me know

Reply:
test cfg file i am not sure we need to change i think more code base

Reply:
Okay, I get your point @PraveenPenguin about the idea of `have script placed in a predefined folder`. Is it in wrapper itself ?

Reply:
ah I think so we can have a local copy (user can have in their fork and populated with his script and he/she can add his scripts, this idea help us to maintain script what we want to use in a global use case   ) so above approach is easy to handle

Reply:
@PraveenPenguin Can you explain a bit more ? Or a run log would help me understand easier.